src/stko/__init__.py

```diff
@@ -1,29 +1,25 @@
+"""Molecular optimisers and property calculators for use with :mod:`stk`."""
+
 from stko import functional_groups, molecule_analysis
 from stko._internal.calculators.extractors.orca_extractor import OrcaExtractor
-from stko._internal.calculators.extractors.xtb_extractor import XTBExtractor
 from stko._internal.calculators.open_babel_calculators import OpenBabelEnergy
 from stko._internal.calculators.orca_calculators import OrcaEnergy
 from stko._internal.calculators.planarity_calculators import (
     PlanarityCalculator,
 )
 from stko._internal.calculators.rdkit_calculators import MMFFEnergy, UFFEnergy
 from stko._internal.calculators.results.energy_results import EnergyResults
-from stko._internal.calculators.results.orca_results import OrcaResults
-from stko._internal.calculators.results.planarity_results import (
-    PlanarityResults,
-)
-from stko._internal.calculators.results.rmsd_results import RmsdResults
-from stko._internal.calculators.results.shape_results import ShapeResults
 from stko._internal.calculators.results.torsion_results import (
     ConstructedMoleculeTorsionResults,
-    TorsionResults,
 )
-from stko._internal.calculators.results.xtb_results import XTBResults
 from stko._internal.calculators.rmsd_calculators import (
     RmsdCalculator,
     RmsdMappedCalculator,
 )
-from stko._internal.calculators.shape_calculators import ShapeCalculator
+from stko._internal.calculators.shape_calculators import (
```

Comment:
What has happened here? Are you importing through the import in shape calculators?